src/style.css

```diff
@@ -1,3 +1,48 @@
 body {
   margin: 0;
+  font-family: 'Roboto', sans-serif;
 }
+
+header {
```

Comment:
style elements using classes and BEM structure instead of tags, the same with `img`, `nav`, `ul`, `li`, `a`